packages/textlint-tester/src/textlint-tester.ts

```diff
@@ -139,19 +144,19 @@ export type TesterInvalid = {
     output?: string;
     ext?: string;
     inputPath?: string;
-    options?: any;
+    options?: TextlintRuleOptions;
     description?: string;
     errors: TesterErrorDefinition[];
 };
 
 export type TestRuleSet = {
     rules: { [index: string]: TextlintRuleModule };
-    rulesOptions: any;
+    rulesOptions: Record<string, TextlintRuleOptions>;
 };
 
 export type TestPluginSet = {
     plugins: { [index: string]: TextlintPluginCreator };
-    pluginOptions: any;
+    pluginOptions: Record<string, TextlintPluginOptions | boolean>;
```

Comment:
Changed pluginOptions type from Record<string, TextlintPluginOptions> to Record<string, TextlintPluginOptions | boolean>

Reason: Plugin options can be either configuration objects or boolean values for enable/disable. The TestConfigPlugin type already supports options?: TextlintPluginOptions | boolean, but TestPluginSet was not aligned with this. This change ensures type consistency between the plugin configuration types and resolves TypeScript compilation errors when boolean values are passed as plugin options.

Impact: Enables proper type checking for both object and boolean plugin configurations without breaking existing functionality.